securitycenter/api/tests/Tests.csproj

```diff
@@ -1,13 +1,13 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <TargetFramework>net6.0</TargetFramework>
     <IsPackable>false</IsPackable>
   </PropertyGroup>
   <ItemGroup>
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.2.0"/>
     <PackageReference Include="xunit" Version="2.4.1"/>
     <PackageReference Include="xunit.runner.visualstudio" Version="2.4.5"/>
-    <PackageReference Include="Google.Cloud.PubSub.V1" Version="2.10.0"/>
+    <PackageReference Include="Google.Cloud.PubSub.V1" Version="3.0.0"/>
```

Comment:
Is this right, actually? I thought we only released a beta of pubsub.

Reply:
Nope, just fixed it, we only released a beta, and somehow I skipped building this one locally.